### ISSUE TITLE
E3686 Additional properties are not allowed ('ScalingConfiguration')

### ISSUE THREAD

**Issue by jeroenpot** [aws-cloudformation/cfn-lint#3366]
### CloudFormation Lint Version

1.3.2

### What operating system are you using?

Ubuntu

### Describe the bug

AWS::RDS::DBCluster has a scalilng configuration option for serverless databases V1 and is defined here
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-scalingconfiguration

### Expected behavior

Do not report that that ScalingConfiguration is not allowed.

### Reproduction template

```yaml

  Db:
    Type: AWS::RDS::DBCluster
    DeletionPolicy: Retain
    UpdateReplacePolicy: Retain
    Properties:
      DatabaseName: 'MyDb'
      ScalingConfiguration:
        AutoPause: true
        SecondsUntilAutoPause: 1200
        MaxCapacity: 64
      Engine: aurora-postgresql
      EngineMode: serverless
      EngineVersion: '13.12'
      MasterUsername: 'x'
      MasterUserPassword: 'y'
      StorageEncrypted: true
      DeletionProtection: true

```
